src/GlyphExtracter.java

```diff
@@ -0,0 +1,101 @@
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.util.ArrayList;
+ 
+public class GlyphExtracter {
+    public ArrayList<BufferedImage> lettersAfterCrop;

```

Comment:
Zapomniałem wcześniej, tutaj w deklaracjach tych list powinien być interfejs (czyli List<BufferedImage>). Tak samo metoda scalpel powinna zwracać List<BufferedImage> (inne elementy systemu nie muszą "wiedzieć", że to konkretnie ArrayList).